app/src/main/res/values/dimens.xml

```diff
@@ -19,7 +19,7 @@
 --><resources>
     <dimen name="window_popup_corner_radius">5dp</dimen>
 
-    <dimen name="default_key_height">48dp</dimen>
+    <dimen name="default_key_height">42dp</dimen>
```

Comment:
This value is arbitrary. If you have strong feelings for this value, then it should be a part of a new theme.

Reply:
"I don't have strong feelings for this value, the current keyboard theme makes it impossible to type, if you want to type freely and fast it makes you miss keys. I make the keys larger and since this value makes it feel non natural I enhance it to be larger in order for the keyboard to feel native."